src/dos/dos_programs.cpp

```diff
@@ -739,7 +737,7 @@ class BOOT : public Program {
 
 		swapInDisks();
 
-		if(imageDiskList[drive-65]==NULL) {
+		if(!imageDiskList[drive-65]) {
```

Comment:
I think this `65` could be replaced by `'A'`?

Reply:
Got them all.